playmusicexporter/src/main/java/re/jcg/playmusicexporter/activities/MusicContainerListActivity.java

```diff
@@ -328,28 +329,22 @@ public boolean onCreateOptionsMenu(Menu menu) {
         MenuInflater inflater = getMenuInflater();
         inflater.inflate(R.menu.music_track_list, menu);
 
-        MenuItem itemRefreshLibrary = menu. findItem(R.id.action_refresh);
-        itemRefreshLibrary.setOnMenuItemClickListener(new MenuItem.OnMenuItemClickListener()
+        MenuItem itemRefreshLibrary = menu.findItem(R.id.action_refresh);
+        itemRefreshLibrary.setOnMenuItemClickListener(item ->
         {
-            @Override
-            public boolean onMenuItemClick(MenuItem item)
+            try
             {
-                try
-                {
-                    mPlayMusicManager.realoadDatabase();
-                }
-                catch (NoSuperUserException e)
-                {
-                    e.printStackTrace();
-                }
-                catch (CouldNotOpenDatabaseException e)
-                {
-                    e.printStackTrace();
-                }
+                mPlayMusicManager.realoadDatabase();
                 mPlayMusicManager = null;
                 loadPlayMusicExporter();
-                return true;
+	            Toast.makeText( this, R.string.databasereloaded, Toast.LENGTH_SHORT).show();
+            }
+            catch (NoSuperUserException | CouldNotOpenDatabaseException e)
+            {
+	            Toast.makeText( this, R.string.dialog_superuser_access_denied_title, Toast.LENGTH_SHORT).show();
```

Comment:
Isn't this quite misleading, if the thrown exception is a CouldNotOpenDatabaseException? I would not expect that to be thrown though, I've never seen it. Still, that might be something we should change later on, even though I wouldn't wait with merging the PR because of that.